source/databricks/calculation-engine/tests/calculation/wholesale/test_tariff_calculators.py

```diff
@@ -339,34 +340,179 @@ def test__calculate_tariff_price_per_ga_co_es__rounds_total_amount_correctly(
     assert actual_amount == expected_total_amount
 
 
-def test__sum_within_month__on_tariff(
+def test__sum_within_month__creates_sum_per_month(
```

Comment:
A number of things are summed but just one is tested here
```suggestion
def test__sum_within_month__sums_amount_per_month(
```